x-pack/plugins/aiops/public/components/change_point_detection/change_point_detection_context.tsx

```diff
@@ -241,11 +241,14 @@ export const ChangePointDetectionContextProvider: FC = ({ children }) => {
       if (requestParamsFromUrl.filters) {
         filterManager.setFilters(requestParamsFromUrl.filters);
       }
+      if (requestParamsFromUrl.query) {
```

Comment:
Is this query getting cleared correctly when the data view is changed. Here I used farequote with a query of `responsetime > 150`, ran the analysis. Then I changed the data view to cloudwatch:

<img width="707" alt="Screenshot 2024-04-23 at 09 33 34" src="https://github.com/elastic/kibana/assets/7405507/3523e2b6-5488-41bf-98df-e7f8938e213b">